mlrun/__main__.py

```diff
@@ -994,6 +994,11 @@ def project(
                 timeout=timeout,
                 overwrite=overwrite_schedule,
             )
+        except mlrun.errors.MLRunConflictError as error:
+            # We don't want to print traceback in this case because it will add the SDK error with "overwrite = True"
+            message = f"failed to run pipeline, {error}".replace("overwrite = True", "--overwrite-schedule")
+            had_error = True
+            print(message)
```

Comment:
Another option:
It can also be inserted into the existing exception block to avoid a bit duplication..

Also, do we need the had_error part in lines 1008-1011? looks like a redundant double check and  it can be inserted into the exception block.